app/helpers/issue_view_columns_issues_helper.rb

```diff
@@ -50,7 +50,14 @@ def render_issue_row(child, level, hidden = false, columns_list, manage_relation
     # Build the content for each table cell
     field_content = content_tag('td', check_box_tag('ids[]', child.id, false, id: nil), class: 'checkbox')
 
-    if child.descendants.any?
+    # If all children are closed and hidden, do not show the expand/collapse button
+    with_closed_issues = (params[:with_closed_issues] == "true")
+    status_column = columns_list.find { |column| column.instance_variable_get(:@name) == :status }
+    all_descendants_closed = child.descendants.all? do |descendant|
+      column_content(status_column, descendant) == "Closed"
```

Comment:
Question: If one of the `descendants` is `Closed` does the cycle stops? Or it continues until it processed all of the?

Reply:
If one of the descendants is not Closed it breaks the loop, if they are all closed it traverses all descendants